mne/label.py

```diff
@@ -580,9 +581,12 @@ def morph(self, subject_from=None, subject_to=None, smooth=5, grade=None,
         data = self.values[:, np.newaxis]
         stc = SourceEstimate(data, vertices, tmin=1, tstep=1,
                              subject=subject_from)
-        stc = morph_data(subject_from, subject_to, stc, grade=grade,
-                         smooth=smooth, subjects_dir=subjects_dir,
-                         warn=False, n_jobs=n_jobs)
+        stc = SourceMorph(subject_from=subject_from,
+                          subject_to=subject_to,
+                          spacing=grade,
+                          smooth=smooth,
+                          subjects_dir=subjects_dir,
+                          warn=False)(stc)
```

Comment:
the n_jobs parameter is not ignored.

why is SourceMorph not supporting n_jobs as before @TommyClausner ?

Reply:
ok I get it now. precomputed is True all the time now so no need for n_jobs